ui/src/layout/Layout.tsx

```diff
@@ -41,12 +30,26 @@ const styles = (theme: Theme) => ({
     },
 });
 
+const localStorageThemeKey = 'gotify-theme';
+type ThemeKey = 'dark' | 'light';
+const themeMap: {[key in ThemeKey]: Theme} = {
```

Comment:
TypeScript has a predefined type for this "map" type.
```suggestion
const themeMap: Record<ThemeKey, Theme> = {
```